corehq/apps/users/management/commands/reset_demo_users_restores.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.ota.utils import reset_demo_user_restore
+from corehq.apps.users.dbaccessors import get_practice_mode_mobile_workers
+from corehq.apps.users.models import CommCareUser
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Reset restores for demo users"
```

Comment:
Might be nice to specify "in a given domain" or something.

Reply:
Thanks! I will update that

Reply:
https://github.com/dimagi/commcare-hq/pull/36156/commits/46eaf47f0348c874d253ae71d3acc3734cb10bd6